CMakeLists.txt

```diff
@@ -14,7 +14,7 @@
 # limitations under the License.
 # ******************************************************************************
 
-cmake_minimum_required (VERSION 3.4)
+cmake_minimum_required (VERSION 3.11)
```

Comment:
Ubuntu 18.04 has CMake 3.10.2: https://packages.ubuntu.com/bionic/cmake  
Ubuntu 16.04 has 3.5.1: https://packages.ubuntu.com/xenial/cmake

Please do not force users to look for non-default CMake versions. Which feature is not covered by previous 3.4 requirement?

/cc @alalek @mshabunin @ilyachur

Reply:
The minimum cmake version bump was required because of this https://cliutils.gitlab.io/modern-cmake/chapters/projects/fetch.html which is supported starting from 3.11

Reply:
Is there alternatives for it?

/cc @ilya-lavrenov

Reply:
Not that I'm aware of. FetchContent was used for the ONNX dependency to be able to cross compile it for ARM platforms that OV has to support.